File: 1470-shuffle-the-array/1470-shuffle-the-array.cpp
class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        int i = 0, j = n;
        vector<int> v;
        while(j<(2*n)){
            v.push_back(nums[i]);
            v.push_back(nums[j]);
            i++;j++;
        }
        return v;
    }
};

File: 1470-shuffle-the-array/1470-shuffle-the-array.js
/**
 * @param {number[]} nums
 * @param {number} n
 * @return {number[]}
 */
var shuffle = function (nums, n) {
    let start = 0;
    let end = n;
    let arr = [];
    for (let i = 0; i < n; i++) {
        arr.push(nums[start]);
        arr.push(nums[end]);
        start += 1;
        end += 1;
    }
    return arr;

}; 